### DISCUSSION TITLE
Untrusted players shouldn't be allowed to damage name tagged hostile mobs inside claims

### DISCUSSION THREAD

**Discussion by WhatsTheBadNews** [GriefPrevention/GriefPrevention#1407] [Ideas]
Name tagged hostile mobs in claimed areas should be immune to outside player damage, this is so farms/grinders/builds that rely on hostile mobs cannot be tampered with.

**Comment by bigpresh** [comment]
I'm not sure that's a good idea - otherwise you could just have a whole bunch of hostile mobs as "guards" which can kill other players but can't be killed by them, which would be a bit unbalancing.

**Comment by Jikoo** [comment]
This has been suggested several times and will not be added to base GP. The addon [GPExtraProtections](https://github.com/GPAddons/GPExtraProtections) is available if you really think it's necessary.
